app/controllers/users_controller.rb

```diff
@@ -1,2 +1,9 @@
 class UsersController < ApplicationController
+
+  def show
+    videos = @current_user.videos.to_ary.sort_by{|video| video.posted_at}.reverse![0..8]
+    tweets = @current_user.tweets.to_ary.sort_by{|tweet| tweet.posted_at}.reverse![0..8]
```

Comment:
I agree with Vika. You could DRY up your code by extracting this repeated sorting logic into a helper method, model method, or private controller method.